Tools/PostProcessing/Visualization.ipynb

```diff
@@ -45,7 +45,9 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "plotfile = './diags/plotfiles/plt00001'\n",
+    "diag_name = 'diag' # E.g., diagnostics.diags_names = diag\n",
+    "iteration = 0\n",
+    "plotfile = './diags/'+diag_name+str(iteration).zfill(5)\n",
```

Comment:
I happen to notice this. A simpler way of expressing this is
`"plotfile = f'./diags/{diag_name}{iteration:05d}'\n",`

Reply:
@dpgrote Thanks Dave! What is the corresponding python code in jupyter notebook? I'm not sure if I get your point.

Reply:
Hi @Yin-YinjianZhao This is the plain Python code:
`plotfile = f'./diags/{diag_name}{iteration:05d}`
This is just a suggested simplification, using the new f-string Python syntax. This new syntax makes writing formatted string much easier and cleaner.

Reply:
@dpgrote I see, good to know this!
I tried it, It worked for my python, but not for an older version of python on my another computer.
I prefer to use the old style, just in case a user (like me) who is not an expert in python having an old version may run into troubles.

Reply:
That is true that it requires a newer version of Python, version 3.6 or newer. For me, this feature alone is useful enough that I only use the newest version anymore.

Reply:
Sounds great to me, too.
I'll put a quick discussion point on our agenda today if we can generally agree on Python 3.6+
(Note that Python<=3.5 is EOL: https://endoflife.date/python).

f-string quick tutorial:
https://realpython.com/python-f-strings/

Reply:
Although there is no currently supported Python that does not support f-strings, this will also work with older python's:
```suggestion
    "plotfile = './diags/{}{:05d}'.format(diag_name, iteration)\n",
```